de.nagstamon.nagstamon.yml

```diff
@@ -0,0 +1,111 @@
+app-id: de.nagstamon.nagstamon
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: nagstamon
+rename-desktop-file: nagstamon.desktop
+rename-icon: nagstamon
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=.nagstamon
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --filesystem=xdg-run/keyring
+  - --filesystem=home
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - name: libsecret
+    buildsystem: meson
+    config-opts:
+      - -Dgcrypt=false
+      - -Dgtk_doc=false
+      - -Dintrospection=false
+      - -Dmanpage=false
+      - -Dvapi=false
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+    sources:
+      - type: archive
+        url: https://gitlab.gnome.org/GNOME/libsecret/-/archive/0.20.5/libsecret-0.20.5.tar.gz
```

Comment:
Unless you need 0.20.5 I suggest using this from shared-modules https://github.com/flathub/shared-modules/blob/master/libsecret/libsecret.json

Reply:
Changed to your proposal.